src/api/app/controllers/source_project_package_meta_controller.rb

```diff
@@ -0,0 +1,81 @@
+class SourceProjectPackageMetaController < SourceController
```

Comment:
Why not putting it in the source namespace we already started? Also I would avoid to inherit from another controller but the ApplicationController to avoid side effects if possible.

Reply:
it was done for a single "easy" controller. In that case, we would go deep in the namespacing like ```Source::Project::Package::Meta```, so I don't know if we will do it consequently